e2e/apprunner/front-end/main.go

```diff
@@ -5,129 +5,68 @@ package main
-	resp, err := http.Get(endpoint)
-	if err != nil {
-		log.Printf("🚨 could call service discovery endpoint: err=%s", err)
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
-	log.Println("Get on ServiceDiscovery endpoint Succeeded")
-	defer resp.Body.Close()
-	body, _ := ioutil.ReadAll(resp.Body)
-	w.WriteHeader(http.StatusOK)
-	w.Write(body)
-}
-
-type Secret struct {
-	Username string `json:"username"`
-	Password string `json:"password"`
-	Host     string `json:"host"`
-	DBName   string `json:"dbname"`
-	Port     int    `json:"port"`
-}
+// ProxyRequest makes a GET request to the query param "url"
+// from the request and writes the response from the url to the response.
+func ProxyRequest(w http.ResponseWriter, r *http.Request) {
+	url := r.URL.Query().Get("url")
+	log.Printf("Proxying request to %q\n", url)
 
-// DBGet calls an aurora DB and returns a timestamp from the database.
-func DBGet(w http.ResponseWriter, req *http.Request) {
-	ctx, cancel := context.WithTimeout(req.Context(), 20*time.Second)
+	ctx, cancel := context.WithTimeout(r.Context(), 5*time.Second)
 	defer cancel()
 
-	cfg, err := config.LoadDefaultConfig(ctx,
-		config.WithRegion(os.Getenv("AWS_DEFAULT_REGION")),
-	)
-	if err != nil {
-		log.Printf("🚨 load aws config: %s", err)
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
+	handleErr := func(format string, a ...any) {
+		msg := fmt.Sprintf(format, a...)
+		log.Println(msg)
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(msg))
 	}
 
-	mgr := secretsmanager.NewFromConfig(cfg)
-	out, err := mgr.GetSecretValue(ctx, &secretsmanager.GetSecretValueInput{
-		SecretId: aws.String(os.Getenv("FRONTENDCLUSTER_SECRET_ARN")),
-	})
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
 	if err != nil {
-		log.Printf("🚨 retrieve aurora secret: err=%s\n", err)
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
-
-	secretValue := aws.ToString(out.SecretString)
-	if secretValue == "" {
-		log.Print("🚨 empty aurora secret value\n")
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+		handleErr("build request: %s", err)
 		return
 	}
 
-	secret := Secret{}
-	if err := json.Unmarshal([]byte(secretValue), &secret); err != nil {
-		log.Printf("🚨 unmarshal rds secret: err=%s\n", err)
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
+	log.Printf("Built request")
 
-	source := fmt.Sprintf("host=%s port=%d user=%s "+
-		"password=%s dbname=%s",
-		secret.Host, secret.Port, secret.Username, secret.Password, secret.DBName)
-	db, err := sql.Open(postgresDriver, source)
+	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
-		log.Printf("🚨 sql open: err=%s\n", err)
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+		handleErr("do request: %s", err)
 		return
 	}
-	defer db.Close()
+	defer resp.Body.Close()
+	log.Printf("Did request")
```

Comment:
nit (and a few other places where `f` isn't needed):
```suggestion
	log.Print("Did request")
```